class-wp-twitter-api.php

```diff
@@ -137,6 +137,7 @@ public function get_access_token( $verifier ) {
 	 */
 	public function get( $url, $params = array(), $defaults = array() ) {
 		$params = wp_parse_args( $params, $defaults );
+		$params = apply_filters( 'wp_twitter_api_query_params', $params, $url );
```

Comment:
Here is some documentation no how to properly add a docblock to a WordPress hook or filter.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
Thanks @kjbenk for the link!  I'll add that.